package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Order_Tracking",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
```

Comment:
It's recommended to provide a brief description of your application. This will help other developers understand what your application does.